#mensaje como una maquina de estados para monitoriar el movimiento de la superficie
#de cada robot, para evitar colision entre ellos

#una variable tipo booleana true/false donde guardamos variable de nombre in_action
bool in_action  # si el robot esta dentro del area de la banda transportadora

#una variable flotante de 64 bits llamada upper_boundary
float64 upper_boundary  # los datos de la superficie del robot


